vue/src/views/Home.vue

```diff
@@ -3,7 +3,7 @@
     <div class="page--home__hero">
       <img alt="RapidSkills Logo" :src="APP_LOGO" class="page--home__hero__logo" />
       <div class="page--home__text page--home__hero__text page--home__hero__text--title">
-        Create new apprenticeship standards in a snap!
+        Create new apprenticeship standards in a snap FOO Dec 23 3:01pm!
```

Comment:
To be removed I assume

Reply:
Yes! Just testing that changes are getting built correctly :)